laboratory-servlet-api-integration/src/it/java/com/wixpress/petri/test/LaboratoryIT.java

```diff
@@ -141,4 +140,13 @@ public void reportsSingleSpecToPetri() throws Exception {
         assertThat(specs.get(0).getScopes(), CoreMatchers.is(Arrays.asList(ValidStubSpecDefinition_1.scopeDefinitions)));
     }
 
+    @Test
+    public void resilientToPetriServerDowntime() throws Exception {
```

Comment:
Re name- 'cachesExperiments' feels a bit problematic to me since it sounds to me like too much about the solution (caching) and not about the need (resiliency AND not incurring the overhead on user thread). 
WDYT about
'doesNotUsePetriServerOnUserRequestAndSoMorePerformantAndResilientToPetriServerDowntime'?
Bombastic and long but is honest and complete